src/Compilers/Core/MSBuildTask/DotnetHost.cs

```diff
@@ -66,7 +66,7 @@ private static string PrependFileToArgs(string pathToTool, string commandLineArg
         {
             var builder = new CommandLineBuilderExtension();
             builder.AppendFileNameIfNotNull(pathToTool);
-            builder.AppendTextUnquoted(commandLineArgs);
+            builder.AppendTextUnquoted(" " + commandLineArgs);
```

Comment:
I think we should a test to verify this.

Reply:
I just attempted to, but: MSBuildTasksTests is desktop-only, and this requires CoreCLR to hit this code path and validate. :')

Reply:
Ah, right